CMakeLists.txt

```diff
@@ -112,27 +112,14 @@ add_library (acquilib SHARED
   src/capture.cpp
   src/camera.cpp
 )
-
 add_dependencies(acquilib ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS} ${PROJECT_NAME}_gencfg)
 target_link_libraries(acquilib ${LIBS} ${catkin_LIBRARIES})
 
-add_library(capture_nodelet src/capture_nodelet.cpp src/subscriber_nodelet.cpp)
```

Comment:
I think this was an example of how to write the subscriber, it would be helpful to leave it in there...

Reply:
I actually like the idea of having an example on how to write a nodelet(or how to write a subscriber nodelet).
The way it was tied in to the libcapture_nodelet library is not most optimal way, it needs to be separate thing by it self. 

Infact i'm thinking about either
1. I'll just add simple new cpp example pkg  for nodelets to subscribe to camera msgs and add link to it in readme of this package.
or
2. just add dir called example in the root of this repo and simple nodelet subscriber.
let me know which one you think would be better.

Reply:
I would keep an example in this repo itself. Maybe a separate example package in the folder or something.